app/src/main/kotlin/com/intive/picover/common/validator/TextValidator.kt

```diff
@@ -19,13 +19,16 @@ class TextValidator private constructor(
 	}
 
 	class Builder {
-		private var allowEmpty = true
-		private var allowBlank = true
-		private var maxLength: Int? = null
+		var allowEmpty = true
+		var allowBlank = true
+		var maxLength: Int? = null
 
-		fun allowEmpty(allow: Boolean) = apply { allowEmpty = allow }
-		fun allowBlank(allow: Boolean) = apply { allowBlank = allow }
-		fun maxLength(length: Int) = apply { maxLength = length }
-		fun build() = TextValidator(allowEmpty, allowBlank, maxLength)
+		fun build() =
+			TextValidator(allowEmpty, allowBlank, maxLength)
```

Comment:
minor: Shouldn't this stay as it was :D ?

Reply:
Kotlin 1.0 has a rule for that to keep it in one line, as it was before

Reply:
Done